src/languages/es.js

```diff
@@ -694,6 +694,16 @@ export default {
             setDefaultFailure: 'No se ha podido configurar el método de pago.',
         },
         addBankAccountFailure: 'Ocurrió un error inesperado al intentar añadir la cuenta bancaria. Inténtalo de nuevo.',
+        getPaidFaster: 'Recibe pagos más rápido',
+        addPaymentMethod: 'Agregue un método de pago para enviar y recibir pagos directamente en la aplicación.',
+        getPaidBackFaster: 'Recibe tu dinero más rápido',
+        secureAccessToYourMoney: 'Acceso seguro a su dinero',
+        receiveMoney: 'Recibe dinero en tu moneda local',
+        expensifyWallet: 'Expensify Wallet',
+        sendAndReceiveMoney: 'Envía y recibe dinero desde tu Expensify Wallet.',
+        bankAccounts: 'Cuentas bancarias',
+        addBankAccountToSendAndReceive: 'Agregue una cuenta bancaria para enviar y recibir pagos directamente en la aplicación.',
+        addBankAccount: 'Agregar cuenta bancaria',
```

Comment:
What is the context of this? because I think it should be
`Agregar una cuenta bancaria`

Reply:
If not done yet, translation texts should be requested to the Expensify/Design team or in the open source slack channel